backend/src/main/java/com/sep6/backend/service/ActorsServiceImpl.java

```diff
@@ -13,17 +14,17 @@ public class ActorsServiceImpl implements ActorsService {
     private ActorsRepository repository;
 
     @Override
-    public List<Person> getActorsBySearch(String search) {
+    public List<PersonProjection> getActorsBySearch(String search) {
         return repository.getActorsBySearch(search);
     }
 
     @Override
-    public List<Person> getActors() {
+    public List<PersonProjection> getActors() {
         return repository.getActors();
     }
 
     @Override
-    public Person getActorById(int id) {
-        return repository.getActorById(id).orElseThrow();
+    public PersonMoviesProjection getActorById(int id) {
+        return repository.getActorById(id);
```

Comment:
when the method in the repository returns Optional this method should implement .orElseThrow()

Reply:
Changed